### ISSUE TITLE
Can't patch loop of type <class 'NoneType'>

### ISSUE THREAD

**Issue by adriangonz** [erdewit/nest_asyncio#62]
After updating to `nest_asyncio==1.5.2`, it seems that our test suite has started to fail with the following error (the `nest_asyncio==1.5.1` release still seems to work fine though):

```
runtimes/alibi-explain/tests/conftest.py:29: in <module>
    nest_asyncio.apply()
../../.virtualenvs/mlserver-37/lib/python3.7/site-packages/nest_asyncio.py:14: in apply
    raise ValueError('Can\'t patch loop of type %s' % type(loop))
E   ValueError: Can't patch loop of type <class 'NoneType'>
```

Could this be due to calling `nest_asyncio.apply()` before an async loop has been created?

**Comment by foxik** [comment]
We have also been bitten by the same problem -- after installation of Jupyter, `jupyter-notebook` crashed because `loop` is `None` during its startup.

Downgrade to `nest_asyncio` 1.5.1 solves the problem.

**Comment by erdewit** [comment]
This is fixed now in v1.5.3

I was trying to replace ``asyncio.get_event_loop()`` as it will apparently start to throw an exception in Python 3.11 when the event loop is not running.
